app/services/SSJ/initialize.rb

```diff
@@ -0,0 +1,23 @@
+# initialize the workflow for this user
+
+# from here we cna use api to update
+# keep mapping data here to workflow
+#
+
+class SSJ::Initialize < BaseService
+  def initialize(workflow_definition)
+    # TODO: pass in user
+    @workflow_definition = workflow_definition
+  end
+
+  def run
+    wf_instance = @workflow_definition.instances.create!
+    # TODO: associate to user somehow
```

Comment:
just stamp out the latest "version"?   bc the version might increase after so i'm guessing we just copy it over here at time of instantiation.  double check this though

Reply:
the definition's version shouldn't change though right?

Reply:
the instance belongs to the definition

Reply:
yeah you are right. ignore, thx!